src/pages/_document.tsx

```diff
@@ -14,7 +14,19 @@ class MyDocument extends Document {
       <Html lang="en">
         <Head />
         <body>
+          <div id="nypl-header"></div>
+          <script
+            type="module"
+            src="https://nypl-header-a8fn0wqv5-nypl.vercel.app/header.min.js?containerId=nypl-header"
```

Comment:
When you get a chance, can you update the Vercel URL to the "production" one for the Header and Footer? https://nypl-header-app.vercel.app/ Thanks!